configs/minigpt4/minigpt-4_vicuna-7b_caption.py

```diff
@@ -57,10 +57,18 @@
     task='caption',
     prompt_template='###Human: {} ###Assistant: ',
```

Comment:
I noticed that you have replaced the `prompt_template` with `en_prompt_tempalte` and `zh_prompt_template`, and here haven't been modified.
I think you can also use a dict instead of several arguments to specify multi-language prompt templates, which can be more extensible.

Reply:
Fixed in the above issue.